package.json

```diff
@@ -42,7 +45,12 @@
   },
   "private": true,
   "devDependencies": {
-    "autoprefixer": "^6.3.6"
+    "autoprefixer": "^6.3.6",
+    "babel-eslint": "^7.0.0",
+    "eslint": "^3.8.0",
+    "eslint-config-meteor": "0.0.9",
+    "eslint-plugin-babel": "^3.3.0",
+    "eslint-plugin-meteor": "^4.0.1"
   },
```

Comment:
I believe that another dep we would need is `eslint-import-resolver-meteor` to help ESLint to recognize that the modules imported from meteor packages exist (ex: `import Telescope from 'meteor/nova:lib';`).